apps/app/src/views/routes/NewOrder/components/OrderForm.tsx

```diff
@@ -131,9 +131,9 @@ export const OrderForm = ({
   };
 
   const orderCreationEnabled =
-    typeof fulfillmentConfig[user.org_id]?.sendOrder !== 'undefined'
-      ? fulfillmentConfig[user.org_id]?.sendOrder
-      : fulfillmentConfig.default.sendOrder;
+    typeof fulfillmentConfig[process.env.REACT_APP_ENV_NAME][user.org_id]?.sendOrder !== 'undefined'
```

Comment:
I'm not a TS expert, naturally these are complaining about potentially being undefined, but checking beforehand doesn't work either

<img width="666" alt="image" src="https://user-images.githubusercontent.com/3934326/222238846-36c6034e-6fd9-47ee-b833-f83d16d1b37f.png">

Reply:
if you "know" better than TypeScript that it's always defined, postfix the variable with a ! (e.g. process.env.REACT_APP_ENV_NAME!). If you think it may be undefined at any point - you should wrap it in an if statement so that TypeScript does what is called "type narrowing"

Reply:
is REACT_APP_ENV_NAME ever undefined.

Also, the correct if statement would be:
```
if ('REACT_APP_ENV_NAME' in process.env) {
  ...
}
```

Reply:
it's not, but in my experience react will always consider env variables to potentially be undefined

Reply:
You could add a "!" if to end the of each use (like the above e.g.) if you know they're always defined to avoid the if statement.